src/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -279,12 +272,12 @@
         action.setCallback(this, callback);
         $A.enqueueAction(action);
 
-        // delete any contacts
-        var listConDelete = component.get("v.listConDelete");
-        listConDelete = this.addPrefixToListObjectFields(namespacePrefix, listConDelete);
-        action = component.get("c.deleteContacts");
+        // remove any contacts
+        var listConRemove = component.get("v.listConRemove");
+        listConRemove = this.addPrefixToListObjectFields(namespacePrefix, listConRemove);
```

Comment:
Yes, we are doing an upsert, and if some of the field names are incorrectly namespaced,I assume we'd get an error.  best to be consistent and always correctly fixup the field names before any server call.